reconciler/reconciler.go

```diff
@@ -331,6 +106,24 @@ func (r *Reconciler) wrappedInactiveEnqueue(
 	}
 }
 
+// addToqueueMap adds a *types.AccountCurrency
+// to the prune map at the provided index.
+func (r *Reconciler) addToqueueMap(
+	acctCurrency *types.AccountCurrency,
+	index int64,
+) {
+	r.queueMapMutex.Lock()
+	key := types.Hash(acctCurrency)
+	if _, ok := r.queueMap[key]; !ok {
+		r.queueMap[key] = map[int64]int{}
+	}
+	if _, ok := r.queueMap[key][index]; !ok {
+		r.queueMap[key][index] = 0
+	}
```

Comment:
This is not needed `r.queueMap[key][index]` defaults to `0`. I guess the tradeoff here is `better readability` or `slightly better performance` (holding write lock a little bit shorter)

Reply:
Was being overly protective to avoid a panic here! Good catch (didn't know that about `int` values): https://play.golang.org/p/-Wb6ZJpsQZU